core/src/main/java/com/yahoo/ycsb/measurements/PeriodicHistogram.java

```diff
@@ -0,0 +1,352 @@
+/**
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+package com.yahoo.ycsb.measurements;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.Properties;
+
+import com.yahoo.ycsb.measurements.exporter.MeasurementsExporter;
+
+/**
+ * Take measurements and maintain a histogram of a given metric, such as READ LATENCY.
+ * Print out this histogram at periodic intervals.
+ * 
+ * @author cooperb
```

Comment:
please rely on git authorship information instead of author tags. if you need to refer to multiple authors, just add them in the commit message like

```
Amending-Author: Some Name <sname@example.com>
```